api/opds.py

```diff
@@ -0,0 +1,50 @@
+from nose.tools import set_trace
+from opds.opds_writer import OPDSFeed
+
+class DictionaryFeed(OPDSFeed):
+  SCHEMA_DEFINED_TERM = "http://schema.org/DefinedTerm"
+  TIME_FORMAT = '%Y-%m-%dT%H:%M:%SZ%z'
+
+  def __init__(self, word, url, language, definitions):
+    super().__init__(word, url)
+
+    self.language = language
+    self.feed["definitions"] = []
+
+    self.add_metadata()
+    self.add_definitions(definitions)
+
+  def add_metadata(self):
+    super().add_metadata()
+    self.metadata["title"] = "Definitions for %s" % self.title,
```

Comment:
The comma at the end of this line is why the definition shows up as a tuple. `"foo",` means `("foo",)` in Python.

Reply:
Ah, thanks. It was something I forgot to remove.